tests/reducers/orderEntryReducer.test.js

```diff
@@ -37,15 +37,7 @@ describe('Order Entry Configurations reducer', () => {
         const actualState = orderEntryConfigurations(initialState, fetchConfigurationsSuccess(configurations));
         expect(actualState).toEqual(expectedState);
     });
-    it('should handle `FETCH_ORDER_CONFIG_LOADING`', () => {
```

Comment:
Why delete this?

Reply:
The deleted `it` block calls `fetchConfigurationsLoading` in `orderEntryActions.js` action which is no longer used in the app.